packages/components/src/ConnectionCard/ConnectionCard.js

```diff
@@ -54,29 +56,35 @@ export default function ConnectionCard({
   badges,
 }: Props) {
   const hasWayBack = wayBack && wayBack.length > 0;
+  const paddingStyle = { paddingHorizontal: padding ? 10 : 0 };
```

Comment:
This probably should live under styles, and the logic could be `padding ? styles.paddingHorizontal : styles.noPaddingHorizontal` or something like this.